content/en-us/reference/engine/globals/RobloxGlobals.yaml

```diff
@@ -101,6 +101,9 @@ functions:
         summary: ''
     tags:
       - Deprecated
+    deprecation_message: |
+      This method has been superceded by `Library.task.delay()` and should not
```

Comment:
```suggestion
      This method has been superseded by `Library.task.delay()` and should not
```

Reply:
Just changing these to the correct spelling (for whatever reason in English, "supercede" becomes "superseded".